[teleport-kube-agent] Add debug option and make authToken optional (#6273)
gravitational/teleport: examples/chart/teleport-kube-agent/README.md

You can also optionally set labels for your Kubernetes cluster using the
format `--set "labels.key=value"` - for example: `--set "labels.env=development,labels.region=us-west-1"`

> It's possible to create secret with the auth token beforehand. To do so, run:

Reviewer: ```suggestion
To avoid specifying the auth token in plain text, it's possible to create a secret containing the token beforehand. To do so, run:
```

```sh

> export TELEPORT_KUBE_TOKEN=`<literal token> | base64 -w0`

Reviewer: ```suggestion
export TELEPORT_KUBE_TOKEN=`<auth token> | base64 -w0`
```

export TELEPORT_NAMESPACE=teleport

cat <<EOF > secrets.yaml
---
apiVersion: v1
kind: Secret
metadata:
  name: teleport-kube-agent-join-token

>   namespace: ${TELEPORT_NAMESPACE}

Reviewer: Make the value error if not set.

```suggestion
  namespace: ${TELEPORT_NAMESPACE?}
```

type: Opaque
data:

>   auth-token: ${TELEPORT_KUBE_TOKEN}

Reviewer: Make the value error if not set.

```suggestion
  auth-token: ${TELEPORT_KUBE_TOKEN?}
```

EOF

$ kubectl apply -f secret.yaml

$ helm install teleport-kube-agent . \
  --create-namespace \

>   --namespace ${TELEPORT_NAMESPACE} \

Reviewer: Make the value error if not set.

```suggestion
  --namespace ${TELEPORT_NAMESPACE?} \
```
